cmd/tcpgoon.go

```diff
@@ -0,0 +1,24 @@
+package cmd
```

Comment:
Now that we have this cmd package, not sure if having a separate cmdutil makes sense... what do you think?

Reply:
mmm, this cmd package is focused on cobra commands/subcommands, and cmdutils are utils used there. We could merge them, but also keep separated.
As you prefer, we could merge them if it's clearer to you.

Reply:
Also, I would like to have only "command" files in package cmd, to make it easier to understand.

Reply:
No problem then... i have some doubts the cmdutil is actually a good name (i don't want this to become a wildcard with no sense), but its true is quite isolated from the main() logic